### PR TITLE
Add basic structure

### PR DESCRIPTION
**Pull request by ehmaddd**
In this Pull Request the following features are added :
- Deleted all the boilerplate from Create React App (text, images, styles)
- Created a new directory called components
- Inside components, created a new Calculator.js file
- In Calculator.js, created a React component that matchs the said design
- Implemented the design only (JSX and CSS)
- Developed just the UI of the calculator
- Used functional components
- Created 1 child component
- Imported calculator in top-level component : App.js
- Tested in the browser; it works without any problems
- Followed the Github flow by creating a new feature branch i.e basic-structure
- Checked for Linters and found no error

### CODE REVIEW

File: src/components/calculator.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+const Calculator = () => (
+  <div className="calculator">
+    <div className="grid-container">
+      <div className="grid-item" id="answer-div">
+        <Result />
+      </div>
+      <div className="grid-item light-grey">AC</div>
+      <div className="grid-item light-grey">+/-</div>
+      <div className="grid-item light-grey">%</div>
+      <div className="grid-item orange">+</div>
```
Comment (cynthiainga) [inline src/components/calculator.js]: - [x] Please kindly change this sign, is a division sign and not an addition sign 🙃 
- [x] Also as mentioned in the project requirements the `Calculator.js` component should match the given design which means "colors, font, padding, width,..." 😁 

Your design

Screenshot 2023-04-25 at 17 39 37

Original design

Screenshot 2023-04-25 at 17 39 56

File: public/manifest.json [hunk 2]
```diff
@@ -1,23 +1,6 @@
...
-    },
-    {
-      "src": "logo512.png",
-      "type": "image/png",
-      "sizes": "512x512"
-    }
-  ],
   "start_url": ".",
   "display": "standalone",
   "theme_color": "#000000",
```
Comment (cynthiainga) [inline public/manifest.json]: - [x] As required all the boilerplate that got created by the CRA (*texts*, *images*, and *styles*) should be deleted, which means including `manifest.json` and `robots.txt` files. Please kindly delete them as well 🙃

**Review by cynthiainga — CHANGES_REQUESTED** [review]
Hi @ehmaddd, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- [x] No linters errors ✔️ 
- [x] Correct Gitflow ✔️ 
- [x] Descriptive PR ✔️ 

## Required Changes ♻️

- [x] Please kindly fix the MIT link in your README file, the link is broken 🙏 🙏 
- [x] Kindly check the comments under the review for more details 👇 👇 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._


Cheers, and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/calculator.js (hunk 1 above)
Comment (ehmaddd) [reply src/components/calculator.js]: Adjust everything according to the instructions

File: public/manifest.json (hunk 2 above)
Comment (ehmaddd) [reply public/manifest.json]: Deleted the said file as instructed

**Review by AdnanOlarmmi — APPROVED** [review]
Hi @ehmaddd,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- Nice one fixing required changes.

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**